queries.sql

```diff
@@ -7,3 +7,31 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered = true;
 SELECT * from animals WHERE name != 'Gabumon';
 SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK TRANSACTION;
```

Comment:
* According to the [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Inside%20a%20transaction%20update%20the%20animals%20table%20by%20setting%20the%20species%20column%20to%20unspecified.%20Verify%20that%20change%20was%20made.%20Then%20roll%20back%20the%20change%20and%20verify%20that%20the%20species%20columns%20went) you are to 

> Verify that the change was made.

> roll back the change 

> verify that the species columns went back to the state before the transaction.

* HINTS you can do all these by following these steps 
* After setting species to unspecified, then
* You check to see if species were actually set to unspecified by running `SELECT species from animals;` 
*  Then rollback the transaction and verify that the change was undone by running ``SELECT species from animals;``

Reply:
Hello David, 
Thanks for your review.
Should I create a new pull request to prove that I verified that the changes made are reflected?
Since I did not add the snapshot.

Reply:
No you don't have to create another PR just effect the changes here

Reply:
Please, do you mean I should attach the snapshot to this comment?

Reply:
Kindly write the queries requested and replace your screenshot with your new results

Reply:
Noted with thanks.